### PR TITLE
CTCP-3737: Remove documentation regarding duplicate LRNs

### PR DESCRIPTION
**Pull request by drnaylor-oc**
Also update definition regarding the 8MB uplift.

### CODE REVIEW

File: resources/public/api/conf/2.0/application.yaml [hunk 1]
```diff
@@ -1388,21 +1374,6 @@ components:
...
-        - message
-      type: object
-      properties:
-        code:
-          enum:
-            - CONFLICT
-          type: string
-        message:
-          type: string
-          description: The description of the error.
```
Comment (fmcgrath) [inline resources/public/api/conf/2.0/application.yaml:1405]: Did we miss this for 11 October comms and content?

**Review by fmcgrath — APPROVED** [review]
Looks OK.

File: resources/public/api/conf/2.0/application.yaml (hunk 1 above)
Comment (drnaylor-oc) [reply resources/public/api/conf/2.0/application.yaml:1405]: No, this bit is just the LRN reuse stuff so it's more supplementary to it - I mentioned that the 409 is no longer sent on a duplicate LRN, but an IE056 is. I just forgot to change it here.

Comment (fmcgrath) [reply resources/public/api/conf/2.0/application.yaml:1405]: OK.

**Pull request merged**